api/app/services/ingestor/strategy/word/inspector/word.rb

```diff
@@ -110,9 +111,21 @@ def stylesheet
             get_contents_from_path(html_file).at("//style").to_html
           end
 
-          def text_section_inspectors
+          def spine_ids
+            text_ingestion_source.map do |item|
+              ::Ingestor::Strategy::Gitbook::Inspector::TextSection.new(item, self)
+                                                                   .source_identifier
+            end
+          end
+
+          def text_ingestion_source
```

Comment:
Another case of a singular method returning an array of things. This is a bad code smell. Refactor this if necessary to make your naming more predictable, unless I'm missing some compelling reason to have it be this way.